tests/asyncio_fixture.py

```diff
@@ -38,6 +38,7 @@ def fake_user_events(session):
         content='test event',
         owner_id=user.id,
         location="Here",
+        is_google_event=False,
```

Comment:
Maybe also add it as default value to `create_event`?

Reply:
I did a bit more :D